### ISSUE TITLE
[BUG] After 60.0 update on Quest 3, can't remove Horizon Feed anymore.

### ISSUE THREAD

**Issue by gaveitatry2** [threethan/LightningLauncher#70]
@threethan - I noticed the Quest 3 had a 60.0 update. Now the Horizon Feeds button is back on the taskbar and it keeps opening up. I wanted it to be removed and disabled. I thought maybe the "Disable Feeds" addon I installed was out of date. So I tried to uninstall it (to later reinstall it), but I couldn't even find it to uninstall it. So no matter what it's installed and I guess it is an out of date version because it no longer works.

**Comment by threethan** [comment]
There is no "Disable Feeds" addon. There is, however, an addon that allows you to open LightningLauncher by hovering the horizon feed icon. There is a button, in the addons page, that lets you easily disable/enable the explore app. Disabling the Explore app, then rebooting, will hide the Horizon Feed icon.

For simplicity, the addons panel only shows the uninstall button if an addon is installed _and_ active. There is nothing stopping you from uninstalling any addons yourself.

**Comment by gaveitatry2** [comment]
> There is no "Disable Feeds" addon. There is, however, an addon that allows you to open LightningLauncher by hovering the horizon feed icon. There is a button, in the addons page, that lets you easily disable/enable the explore app. Disabling the Explore app, then rebooting, will hide the Horizon Feed icon.
> 
> For simplicity, the addons panel only shows the uninstall button if an addon is installed _and_ active. There is nothing stopping you from uninstalling any addons yourself.

Please don't close this Issue. I incorrectly stated "Disabled Feeds", but I meant the "Disable Explorer App". It's just that disabling the explorer app is the same thing as disabling the Horizon Feeds app. I have that installed. I have no idea of how to uninstall. The problem is it no longer works after the Ques 60.0 update. So why would you close this Issue when I would gladly like for the "Disable Explorer App" to work again????

**Comment by threethan** [comment]
This is not an addon, it's a button which is just a shortcut to the app info for the expore app.

There is no way to 'uninstall' that, because it's not installing anything in the first place. Re-enabling the explore app is the opposite of disabling this.

**Comment by gaveitatry2** [comment]
> This is not an addon, it's a button which is just a shortcut to the app info for the expore app.
> 
> There is no way to 'uninstall' that, because it's not installing anything in the first place. There's also no 'version' - it is a button which opens an intent. Re-enabling the explore app is the opposite of disabling it.

I have no idea what you are going on about. I want the Horizon Feeds AKA Explorer button to not show up on the taskbar or whatever you freaking call that bar that has apps and games pinned to it. And I want the Horizon Feeds AKA Explorer window to not open up when I turn my headset on, or close out of a game/app. This feature, which you seem to know so much about, was working before the Quest 60.0 update. Now it isn't working for me. So what is it exactly that you are trying to tell me? Because after a bunch of posts from you, it seems like you can't even acknowledge that it is no longer working or that you have any intention of trying to make it work again. If you don't care about your launcher's features breaking and you don't care to fix it, then just write "I don't care and I'm not going to fix it" instead of wasting my time with these idiotic and pointless posts of yours.

**Comment by gaveitatry2** [comment]
I will come back to you on this. I don't think you will be satisfied until you have given me the biggest headache possible, wasted as much of my time as possible, and blow as much smoke out your behind as possible.

I uninstalled the app for now. I took some screenshots that I will share with you at a later time. I will eventually reinstall the app from Sidequest again, and give it another week or so to test it out to see if the grouping is still broken.

I'm not in a hurry to do it because I haven't felt like using VR this week and also just now I restarted my headset (even though I've restarted it yesterday and the day before AFTER the 60.0 update), and suddenly my room boundary is messed up. And I don't feel like redrawing it again because it covers multiple rooms and it takes time to draw.

Anyway, thanks for nothing. Will message you about this again in a couple of weeks.
